source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,185 @@
+
+```jsx
+{
+  "retries": 1
+}
+```
+
+However, in the event you need to define unique retry attempts for the different modes of Cypress, you can pass the `retries` option an object with the following options:
+
+- `runMode` allows you to define the number of test retries when running `cypress run`
+- `openMode` allows you to define the number of test retries when running `cypress open`
+
+```jsx
+{
+  "retries": {
+    // Configure retries for `cypress run` 
+    // Default is 2
+    "runMode": 1,
+    // Configure retries for `cypress open`
+    // Default is 0
+    "openMode": 3
+  }
+}
+```
+
+### Custom Configurations
+
+#### Individual Test(s)
+
+In the event a specific test requires a custom number of test retries configured, this can be achieved by using the test's local configuration option.
+
+```jsx
+// Customizing retries for an individual test
+describe('User sign-up and login', () => {
+  // `it` test block with no custom configuration
+  it('should redirect unauthenticated user to sign-in page', () => {
+    ...
+  })
+
+  // `it` test block with custom configuration
+  it('allows user to login', { 
+    retries: {
+      runMode: 3,
+      openMode: 2
+    }
+  }, () => {
+    ...
+  })
+})
+
+```
+
+#### Test Suite(s)
+
+If you want a suite of tests to re-run in requires a custom number of test retries configured, this can be achieved by using the test's local configuration option
+
+```jsx
+// Customizing retries for a suite of tests
+describe('User bank accounts', {
+  retries: {
+    runMode: 3,
+    openMode: 1,
+  }
+}, () => {
+  // Individual tests will be run normally
+  it('allows a user to view their transactions, () => {
+    ...
+  }
+
+  it('allows a user to edit their transactions, () => {
+    ...
+  }
+})
+
+```
+
+You can find more information about custom configurations here: {% url "Test Configuration" configuration#Test-Configuration %}
+
+## Assets
+
+With test retries, Cypress will now generate any assets (i.e., screenshots, video recordings, etc.) per each test attempt. The assets will be properly labeled with the attempt number appended at the end of each asset.
+
+    🖼 Insert screenshot
+
+## Tips and Strategies
+
+While test retries are great for helping to avoid false negatives from failing an entire test run, it is not a good replacement for writing good tests. As a result, here are some tips and strategies to keep in mind in order to maximize the effectiveness of your tests with test retries:
+
+- If you are noticing that you need to increase the number of retries, this cause is more likely due to how the tests are written and is worth spending the time to investigate
+- If you use `Run all specs` a lot in `cypress open` mode, make sure to configure your Cypress instance to have a global `retries` of `2` so you can simulate what is being run in `cypress run` mode.
```

Comment:
I don't think this is a good suggestion honestly and I'm interested to hear the motivation behind why this is suggested.

We don't want people using 'Run all specs' in general. Global hooks **will run differently** in 'Run all specs' as opposed to `cypress run`, so the test retries **will run differently** if they turn them on for both run and open mode and do 'Run all specs' versus `cypress run`, which I think will make things more difficult for them to debug.

Reply:
That's a fair point! I wasn't aware that there using the "Run all specs" was considered an anti-pattern. I'll go ahead and remove it. Thanks for clarifying!